modules/internal/OwnedObject.chpl

```diff
@@ -240,6 +240,17 @@ module OwnedObject {
        that takes over ownership from `src`. `src` will
        refer to `nil` after this call.
      */
+    proc init=(type ThisType, pragma "leaves arg nil" pragma "nil from arg" ref src:_owned) {
+      // Use 'ThisType.chpl_t' in case RHS is a subtype
+      this.chpl_t = ThisType.chpl_t;
+      this.chpl_p = src.release();
+    }
+
+    proc init=(type ThisType, src : _nilType) {
+      this.init(ThisType.chpl_t);
+    }
+
+    // Copy-init implementation to allow for 'new _owned(foo)' in module code
```

Comment:
same here

Reply:
I'm not sure we want to continue calling this one though.... do you know where it is called?

Reply:
I think there are some 'new _owned' in this module, not sure about other modules though.